src/main/java/io/split/android/client/SplitClientConfig.java

```diff
@@ -81,7 +86,10 @@ private SplitClientConfig(String endpoint,
                               int eventsPerPush,
                               long eventFlushInterval,
                               String trafficType,
-                              boolean synchronizeInBackground) {
+                              boolean synchronizeInBackground,
+                              long backgroundSyncPeriod,
+                              boolean backgroundSyncWhenBatteryNotLow,
+                              boolean backgroundSyncWhenWifiOnly) {
```

Comment:
this param is not used inside the constructor. maybe add `_backgroundSyncWhenWifiOnly = backgroundSyncWhenWifiOnly;`